### PR TITLE
[v15.5] Deprecate react-addons-update

### PR DESCRIPTION
**Pull request by koba04**
This PR is for adding deprecation warnings for `react-addons-update`, which is listed in #8854.

### CODE REVIEW

File: src/addons/__tests__/update-test.js [hunk 1]
```diff
@@ -15,6 +15,18 @@ var update = require('update');
 
 describe('update', () => {
 
+  it('warns once when using ReactTransitionGroup', () => {
```
Comment (acdlite) [inline src/addons/__tests__/update-test.js]: Wrong test name

**Comment by koba04** [comment]
@acdlite Thanks! Fixed it!

**Comment by gaearon** [comment]
Hey, sorry we didn’t take those 😞 .

15.5 release was very hard because we realized we couldn’t do it from master because it diverged too much. So we mostly did it manually.

**Comment by koba04** [comment]
No problem! Thank you for your great work!
I'm surprised that v15.5 was released last week. Because I thought it seems to take more time to do it. Awesome!

**Pull request closed without merging**